gitpod-io/website
# [ref-arch/eks] Add external-dns service account to eksctl service account list

## Description

The [external-dns pull request](https://github.com/gitpod-io/website/pull/2649/files) was targeted against the PoV architecture as that's what was under development when the external-dns functionality was added. This commit adds the necessary IAM service account to the EKS single-cluster configuration to allow the external-dns service account/pod to auth against the AWS API.

## Related Issue(s)

See also: https://github.com/gitpod-io/website/pull/2649/files

## How to test

- Create a cluster with the given eksctl configuration
- Install the external-dns helm chart
- Install gitpod and confirm that the gitpod DNS entries were automatically created.

In terms of actual testing this has been run through the wringer pretty thoroughly, shouldn't need actual functional review.

## Release Notes

```release-note
[reference-architecture] Add external-dns service account to EKS single-cluster eksctl config file
```

## Documentation



<a href="https://gitpod-staging.com/#https://github.com/gitpod-io/website/pull/2680"><img src="https://gitpod-staging.com/button/open-in-gitpod.svg"/></a>



<a href="https://gitpod.io/#https://github.com/gitpod-io/website/pull/2680"><img src="https://gitpod.io/button/open-in-gitpod.svg"/></a>
